pkg/watch/health.go

```diff
@@ -131,6 +132,10 @@ func updatePods(
 	for _, pod := range current {
 		inReality := false
 		for _, man := range reality {
+			if man.PodUniqueKey != nil {
+				// We don't health check uuid pods
```

Comment:
I'd worry that separating the decision could result in a situation where health checking is on for a uuid pod, in which case it will override the health of a legacy pod (because we just a health namespace by node/pod id)